static/src/javascripts/projects/common/modules/identity/sign-in-gate/helper.js

```diff
@@ -253,6 +253,20 @@ export const isInvalidSection = (include: Array<string> = []): boolean => {
         }, false);
 };
 
+// hide the sign in gate for certain tags on the site
+// add to the include parameter array if there are specific tags that should be included/overridden
+export const isInvalidTag = (include: Array<String> = []): boolean => {
```

Comment:
I'm not sure I understand what `include` is used for, have I missed something?

Reply:
I took it from `isInvalidSection` but I believe it's to force include a section/tag for the sign in gate
Assuming you have `include = ['help']`, then you would have
```javascript
invalidSection.filter(el => !['help'].includes(el))
invalidSection.filter(('help') => !['help'].includes('help'))
invalidSection.filter(('help') => !(true))
invalidSection.filter(('help') => false)
['about', 'info', 'membership',  'guardian-live-australia', 'gnm-archive']
```
and so you would still show sign in for a 'help' section page where you've added 'help' to `include`

Reply:
With that in mind, I'm not sure when or how you would specify this and it appears to be missing on DCR as an argument to [the equivalent functions](https://github.com/guardian/dotcom-rendering/blob/aa67bc00ed1807f52c8d0151fae80e1188392f3a/src/web/components/SignInGate/displayRule.ts#L50-L74)

Reply:
yeah, i think it might be worth removing the `include` functionality

the idea originally was to be able to show it on certain pages that had been previously excluded, e.g. help section

but we never used this functionality at all (which is why we didn't include it in dcr), and we can always add it back in if it's needed